.gitignore

```diff
@@ -1,4 +1,5 @@
 /vendor
+.idea
```

Comment:
This change seems unrelated to the PR

Reply:
Yes,It's unrelated to the PR,I just get used to add this.

Reply:
Editor settings should be globally ignored :) https://help.github.com/en/articles/ignoring-files#create-a-global-gitignore

Reply:
Thanks,I don't know that.